doc/importing-template.md

```diff
@@ -0,0 +1,59 @@
+## How to import the template into Azure DevOps
+
+### Pre-requisites:
+
+[Create a DevOps Organizaton](https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/create-organization?view=azure-devops#create-an-organization) or use an existing one. 
+
+### Process:
+
+1. Navigate to the [Azure DevOps Demo Generator](https://azuredevopsdemogenerator.azurewebsites.net/Account/Verify) and sign into your DevOps organization.
+2. Fill in **Project Name**, select your Organization, and select **Choose Template**.
+3. Select **Private** and upload [this zip file](/artifacts/iac-migration.zip).
```

Comment:
Some suggestions for Step 3:
3. Download [this zip file](/artifacts/iac-migration.zip) to your local Downloads folder.
4. Select **Private** from the navigation bar and click `Choose File`, navigate to Downloads to upload `iac-migration.zip` to the template. Click **Submit**.